### ISSUE TITLE
You are not setup to receive mobile confirmations

### ISSUE THREAD

**Issue by AndriiSeeker** [Jessecar96/SteamDesktopAuthenticator#954]
Could you please tell me why I don't see any confirmations?
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/01706fba-8f6a-4fcc-bdcf-1374731a4a32)

**Comment by dyc3** [comment]
Check your steam account here: https://store.steampowered.com/account/

It should look like this:

![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/1808807/aca6d031-3b4d-4d85-85ea-a57c7f054e33)

If it doesn't that means you that there is no authenticator linked to the account.

**Comment by AndriiSeeker** [comment]
Thanks, it protected by email
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/8f241545-7ad2-4e97-bb3c-b9a87ea0cde3)

But as i understand, it should be by mobile. Because in SDA i have a confirmation token.
So what should i do? :(

**Comment by AndriiSeeker** [comment]
![image](https://github.com/Jessecar96/SteamDesktopAuthenticator/assets/113901127/9d7c5a78-9f56-4143-98cc-ef056d629db0)
Maybe I need to change it here?

**Comment by dyc3** [comment]
No, you need to set up the authenticator again (which, as reported in other issues, is broken in SDA right now). Your current maFile for this account is invalid.
